cmd/webhook/kodata/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Can you please change the `LICENSE` and the `VENDOR-LICENSE` files to be [symbolic links](https://github.com/tektoncd/triggers/blob/master/cmd/controller/kodata/LICENSE) (see [here](https://github.com/tektoncd/triggers/tree/master/cmd/controller/kodata) for reference).

Reply:
Done